traveler-ios-ui/TravelerKitUI/ViewControllers/ParkingQueryFormViewController.swift

```diff
@@ -39,8 +39,8 @@ public class ParkingQueryFormViewController: UIViewController {
     public override func viewDidAppear(_ animated: Bool) {
         parkingFilterContext?.addObserver(self)
     }
-
-    public override func viewWillDisappear(_ animated: Bool) {
+    
+    deinit {
```

Comment:
Added in `viewDidAppear`, remove in `viewWillDisappear`.  Otherwise, need to add in `viewDidLoad` and remove in `deinit`